src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,18 @@
+package calculator;
+
+public class Calculator {
+
+    private final Validator validator;
+
+    public Calculator() {
+        this.validator = new Validator();
+    }
+
+    public int sum (String [] tokens){
+        int sum = 0;
+        for (int i = 0; i < tokens.length; i++) {
+            sum += validator.check(tokens[i]);
+        }
+        return sum;
```

Comment:
```suggestion
        int sum = 0;
        
        for (int i = 0; i < tokens.length; i++) {
            sum += validator.check(tokens[i]);
        }
        
        return sum;
```
함수 블록 내부에서 공백라인을 활용하여 선언 / 실행 / 결과에 대해 문맥을 나누어 보는건 어떨까요 ?

Reply:
확실히 공백라인을 활용하니까 가독성이 좋아지는 것 같습니다! 수정하겠습니다